### PR TITLE
Test commands

### PR DESCRIPTION
**Pull request by DeveloperInProgress**

### CODE REVIEW

File: Acala/acala-evm-starter/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "start:docker": "docker-compose pull && docker-compose up --remove-orphans",
     "dev": "subql codegen && subql build && docker-compose pull && docker-compose up --remove-orphans",
     "prepack": "rm -rf dist && npm run build",
-    "test": "jest"
+    "test": "subql-node test"
```
Comment (stwiname) [inline Acala/acala-evm-starter/package.json:12]: Did we want this to include the build step first?

**Review by stwiname — COMMENTED** [review]
Nice!

@DeveloperInProgress can you please add an example test to each of the projects. They should all be the same test just a different block number.

File: Acala/acala-evm-starter/docker-compose.yml [hunk 2]
```diff
@@ -31,10 +31,11 @@ services:
       DB_PORT: 5432
     volumes:
       - ./:/app
-    command:
+    command: 
+      - ${SUB_COMMAND} # set SUB_COMMAND env variable to "test" to run tests
       - -f=/app
+      - --disable-historical
```
Comment (stwiname) [inline Acala/acala-evm-starter/docker-compose.yml:37]: Lets remove this flag please. We want historical to be enabled.

File: Acala/acala-evm-starter/docker-compose.yml [hunk 3]
```diff
@@ -31,10 +31,11 @@ services:
       DB_PORT: 5432
     volumes:
       - ./:/app
-    command:
+    command: 
+      - ${SUB_COMMAND} # set SUB_COMMAND env variable to "test" to run tests
```
Comment (stwiname) [inline Acala/acala-evm-starter/docker-compose.yml:35]: Does this work if nothing is set?

File: Acala/acala-evm-starter/package.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
     "start:docker": "docker-compose pull && docker-compose up --remove-orphans",
     "dev": "subql codegen && subql build && docker-compose pull && docker-compose up --remove-orphans",
     "prepack": "rm -rf dist && npm run build",
-    "test": "jest"
+    "test": "subql build && subql-node test"
```
Comment (stwiname) [inline Acala/acala-evm-starter/package.json:12]: We need to add `@subql/node` as a dependency

File: Acala/acala-evm-starter/docker-compose.yml (hunk 3 above)
Comment (DeveloperInProgress) [reply Acala/acala-evm-starter/docker-compose.yml:35]: yes

**Review by stwiname — APPROVED** [review]

**Pull request merged**